Exiled.API/Features/Items/Ammo.cs

```diff
@@ -16,6 +16,14 @@ namespace Exiled.API.Features.Items
     /// </summary>
     public class Ammo : Item
     {
+        /// <summary>
+        /// Specifies the absolute maximum amount of ammo that may be held at one time, if ammo is forcefully given to the player (regardless of worn armor or server configuration).
+        /// <para>
+        /// For accessing the maximum amount of ammo that may be held based on worn armor and server settings, see <see cref="Player.GetAmmoLimit(ItemType)"/>.
+        /// </para>
+        /// </summary>
+        public const ushort AmmoLimit = 65535;
```

Comment:
what do you think about 
=> ushort.MaxValue

Reply:
I agree this should be a property